transport/src/bin/bootnode.rs

```diff
@@ -118,7 +118,7 @@ async fn main() -> anyhow::Result<()> {
         swarm.add_external_address(public_addr);
     }
 
-    swarm.behaviour_mut().kademlia.set_mode(Some(Mode::Server));
+    swarm.behaviour_mut().kademlia.set_mode(Some(Mode::Server)); // Redundant if you are setting an external address.
```

Comment:
I want to keep it for local docker-compose tests, where we don't set any external addresses.